compaction/compaction.cc

```diff
@@ -514,6 +515,9 @@ class compaction {
         _contains_multi_fragment_runs = std::any_of(_sstables.begin(), _sstables.end(), [&ssts_run_ids] (shared_sstable& sst) {
             return !ssts_run_ids.insert(sst->run_identifier()).second;
         });
+        if (_owned_ranges) {
+            _owned_ranges_checker.emplace(*_owned_ranges);
+        }
```

Comment:
Is it just for this? If so it should be moved to the initializer list.

Reply:
done in "compaction: initialize _owned_ranges_checker"